### PR TITLE
fixes for consul in puppet

### PR DESCRIPTION
**Pull request by phrawzty**
@rhelmer `r?`

### CODE REVIEW

File: puppet/modules/socorro/files/etc_consul/ui.json [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
```
Comment (phrawzty) [inline puppet/modules/socorro/files/etc_consul/ui.json]: The new `consul-ui` package ships with this file so we don't need to supply it here.

**Comment by rhelmer** [comment]
r+

File: puppet/modules/socorro/files/etc_consul/common.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
     "data_dir": "/var/lib/consul",
-    "log_level": "INFO",
```
Comment (phrawzty) [inline puppet/modules/socorro/files/etc_consul/common.json]: This is the reason humans aren't supposed to edit JSON by hand. :black_medium_square:

File: terraform/variables.tf [hunk 3]
```diff
@@ -17,7 +17,7 @@ variable "ssh_key_name" {
 }
 variable "base_ami" {
     default = {
-        us-west-2 = "ami-2fbc931f"
+        us-west-2 = "ami-51604e61"
```
Comment (phrawzty) [inline terraform/variables.tf]: I have already built a new base AMI from this PR. :smile:

Comment (rhelmer) [inline terraform/variables.tf]: AMIception

**Pull request merged**